handlers/map-tactile-svg/map-svg.py

```diff
@@ -185,6 +193,63 @@ def handle():
                 g.append(p)
         svg.append(g)
 
+    if "ca.mcgill.a11y.image.preprocessor.nominatim"\
+            in preprocessor:
+        targetData = preprocessor[
+                                 "ca.mcgill.a11y.image.preprocessor.nominatim"]
+        try:
+            latitude = (
+                        (float(targetData["lat"]) - lat_min)
+                        * scaled_latitude)
+            longitude = (
+                        (float(targetData["lon"]) - lon_min)
+                        * scaled_longitude)
+            svg.append(
+                        draw.Circle(
+                                    longitude,
+                                    latitude,
+                                    20,
+                                    fill='green',
+                                    stroke_width=1.5,
+                                    stroke='green',
+                                    aria_label=targetData["name"]))
+
+            """
+            ## Using bounding box occasionally results in the whole map
+            ## being occupied by the target POI
+            ## e.g. when McGill University is the detected target POI
+            bb=targetData["boundingbox"]
+            logging.debug(", ".join(bb))
+            lat_start = (
+                        (float(bb[0]) - lat_min)
+                        * scaled_latitude)
+            height = (
+                        (float(bb[1]) - lat_min)
+                        * scaled_latitude
+                        - lat_start)
+            lon_start = (
+                        (float(bb[2]) - lon_min)
+                        * scaled_longitude)
+            width = (
+                        (float(bb[3]) - lon_min)
+                        * scaled_longitude
+                        -lon_start)
+            svg.append(
+                        draw.Rectangle(
+                                        lon_start,
+                                        lat_start,
+                                        width,
+                                        height,
+                                        fill='red',
+                                        stroke_width=1.5,
+                                        stroke='red',
+                                        aria_label=targetData["name"]))
+            """
+            renderingDescription = "Tactile rendering of map centered at "\
+                + targetData["name"]
+        except KeyError:
+            logging.debug("Reverse geocode data unavailable")
```

Comment:
Highly recommend including the actual key that was not present - this doesn't provide much info as is!

Reply:
Included info on missing key in recent commit and tested for {"latitude": -24.3491146, "longitude": -128.3472909} (by an island in the Pacific ocean).
The resulting log is as follows:
```
image-map-tactile-svg-handler-1  | DEBUG:root:Received request
image-map-tactile-svg-handler-1  | DEBUG:root:Missing key 'lat' in neonatim preprocessor
image-map-tactile-svg-handler-1  | DEBUG:root:Reverse geocode data not added to response
image-map-tactile-svg-handler-1  | DEBUG:root:Sending response
```
Also, a blank SVG is returned (Is this the ideal response?)